test/content/audio/approaching_test.exs

```diff
@@ -19,14 +19,12 @@ defmodule Content.Audio.ApproachingTest do
                {:canned, {"110", spaced(["896", "915", "920", "910", "21014"]), :audio_visual}}
     end
 
-    test "Returns nil for Green Line trips" do
+    test "Returns params for Green Line trips" do
       audio = %Approaching{destination: :riverside, route_id: "Green-D"}
-      assert Content.Audio.to_params(audio) == nil
-    end
 
-    test "Returns nil when destination is Ashmont on the Mattapan line" do
```

Comment:
Do you remember why this special case existed? Or was it not really a special case because we weren't playing Approaching on the Mattapan line at all?

Reply:
This was part of the general suppression of approaching messages for light rail. I assume the test was to verify the distinction between Ashmont RL and Ashmont Mattapan.